.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
+.python-version
+.idea
```

Comment:
Oh and I'd rather leave this sort of .gitignore addition out of this repository. While I totally use pycharm, there isn't any reason to assume what IDE / editor contributors may be using. For gitignore entries unique to your development environment, I recommend simply using a "global" gitignore file instead: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Would you mind backing out these changes?